src/k8s-job-exit-spec/config/k8s-job-exit-spec.yaml

```diff
@@ -482,6 +482,20 @@ spec:
     - reasonRegex: '(?i)^UnexpectedPredicateFailureType$'
       messageRegex: '(?ms).*'
 
+- code: -1007
+  phrase: UnrecoverableSchemaSynchronizeError
```

Comment:
Adjust to below
###########################################################################
# Range: [-1099, -1000]
# Owner: PAI_FC
# Description: PAI_K8S issued failures
###########################################################################5291 



###########################################################################
# Range: [-1199, -1100]
# Owner: PAI_DB_CONTROLLER
# Description: PAI_K8S issued failures
###########################################################################5291

Reply:
done. Use -1100 as the exit code.